auth0/v3/authentication/passwordless.py

```diff
@@ -103,3 +171,17 @@ def sms_login(self, client_id, phone_number, code, scope='openid'):
                 'scope': scope,
             }
         )
+    
+    def sms_login_legacy(self, client_id, phone_number, code, scope='openid'):
```

Comment:
The passwordless login functionality is already covered by `sms_login`. What's this method for?

Reply:
I went through this PR again. Let's remove this method `sms_login_legacy` and keep `sms_login` for compatibility purposes.

From now on, the documented way to consume the passwordless endpoint should be through the `login` method that you added.